site/docs/master/customize-installation.md

```diff
@@ -42,6 +43,16 @@ By default, `velero install` does not install Velero's [restic integration][3].
 
 If you've already run `velero install` without the `--use-restic` flag, you can run the same command again, including the `--use-restic` flag, to add the restic integration to your existing install.
 
+## Enable features
+
+New features in Velero will be released as beta features behind feature flags which are not enabled by default. These features can be enabled using the `--features` flag to the `velero install` command. This flag takes as value, a comma separated list of feature flags to enable. As an e.g. [CSI snapshotting of PVCs][10] can be enabled using `--features=EnableCSI` flags to the `velero install` command.
```

Comment:
```suggestion
New features in Velero will be released as beta features behind feature flags which are not enabled by default. These features can be enabled using the `--features` flag to the `velero install` command. This flag takes as value a comma separated list of feature flags to enable. As an example, [CSI snapshotting of PVCs][10] can be enabled using `--features=EnableCSI` flags in the `velero install` command.
```

Reply:
It's either "`in` or `on` the velero install command" (I always get that mixed up).

Reply:
done!